primitives/comb.fil

```diff
@@ -143,19 +143,19 @@ extern "comb.sv" {
     @[G, L] out: 1,
   ) where L > G, #WIDTH > 0;
 
-  comp ShiftLeft[#WIDTH]<G: L-(G), ?L: 1=G+1>(
+  comp ShiftLeft[#WIDTH, ?#SHIFT_WIDTH=#WIDTH, ?#OUT_WIDTH=#WIDTH]<G: L-(G), ?L: 1=G+1>(
```

Comment:
Looks like this depends on #181 to be merged so let's merge that first before merging this.